docs/design/http_api_swagger.md

```diff
@@ -583,7 +583,9 @@ Status: Internal Server Error
 
 | Name | Type | Go type | Required | Default | Description | Example |
 |------|------|---------|:--------:| ------- |-------------|---------|
-| authFile | string| `string` |  | | Image registries authfile created by executing `podman login` or `docker login` (i.e. ~/.docker/config.json). https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#log-in-to-docker-hub describes how the file can be created and how it is structured. |  |
+| authFile | [interface{}](#interface)| `interface{}` |  | | Image registries authfile created by executing `podman login` or `docker login` (i.e. ~/.docker/config.json).
```

Comment:
The rendered MD formatting seems broken here - probably due to swagger.yaml formatting changes.

Reply:
uh right! I accidentally deleted `type: string` from `swagger.yaml`

Reply:
It didn't help; it's still ill-formatted.